{
	"about": "https://jvndb.jvn.jp/ja/contents/2023/JVNDB-2023-007632.html",
	"title": "django-ses project の django-ses におけるデジタル署名の検証に関する脆弱性",
	"link": "https://jvndb.jvn.jp/ja/contents/2023/JVNDB-2023-007632.html",
	"description": "django-ses project の django-ses には、デジタル署名の検証に関する脆弱性が存在します。",
	"identifier": "JVNDB-2023-007632",
	"references": [
		{
			"text": "https://www.cve.org/CVERecord?id=CVE-2023-33185",
			"source": "CVE",
			"id": "CVE-2023-33185"
		},
		{
			"text": "https://nvd.nist.gov/vuln/detail/CVE-2023-33185",
			"source": "NVD",
			"id": "CVE-2023-33185"
		},
		{
			"text": "https://github.com/django-ses/django-ses/blob/3d627067935876487f9938310d5e1fbb249a7778/CVE/001-cert-url-signature-verification.md",
			"source": "関連文書",
			"id": "github.com (001-cert-url-signature-verification.md)"
		},
		{
			"text": "https://github.com/django-ses/django-ses/commit/b71b5f413293a13997b6e6314086cb9c22629795",
			"source": "関連文書",
			"id": "github.com (b71b5f413293a13997b6e6314086cb9c22629795)"
		},
		{
			"text": "https://github.com/django-ses/django-ses/security/advisories/GHSA-qg36-9jxh-fj25",
			"source": "関連文書",
			"id": "github.com (GHSA-qg36-9jxh-fj25)"
		},
		{
			"text": "https://cwe.mitre.org/data/definitions/347.html",
			"id": "CWE-347",
			"title": "デジタル署名の不適切な検証(CWE-347)"
		}
	],
	"cpe": [
		{
			"text": "cpe:/a:django-ses_project:django-ses",
			"version": "2.2",
			"vendor": "django-ses project",
			"product": "django-ses"
		}
	],
	"cvss": [
		{
			"version": "3.0",
			"score": "5.4",
			"type": "Base",
			"severity": "Medium",
			"vector": "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:L/I:L/A:N"
		}
	],
	"date": "2023-11-24T17:10+09:00",
	"issued": "2023-11-24T17:10+09:00",
	"modified": "2023-11-24T17:10+09:00"
}
